com.unity.render-pipelines.high-definition/Editor/Material/ShaderGraph/LightingSubTarget.cs

```diff
@@ -93,6 +94,15 @@ public override void GetFields(ref TargetFieldContext context)
             // Double Sided
             context.AddField(DoubleSidedFlip,                      systemData.doubleSidedMode == DoubleSidedMode.FlippedNormals && context.pass.referenceName != "SHADERPASS_MOTION_VECTORS");
             context.AddField(DoubleSidedMirror,                    systemData.doubleSidedMode == DoubleSidedMode.MirroredNormals && context.pass.referenceName != "SHADERPASS_MOTION_VECTORS");
+
+            // We need to grab the emission block to check if it is connected, or the default value is non null.
+            bool emissionPassEnabled = false;
+            if (!context.connectedBlocks.Contains(BlockFields.SurfaceDescription.Emission))
```

Comment:
I haven't take care up to know, but is this code done for all shader or just lit shader?  We must generate EmissiveForward pass only for lit shader.